lib/token-translator.js

```diff
@@ -191,10 +189,10 @@ TokenTranslator.prototype = {
          * @returns {void}
          * @private
          */
-        function translateTemplateTokens() {
-            tokens.push(convertTemplatePart(that._tokens, that._code));
-            that._tokens = [];
-        }
+        const translateTemplateTokens = () => {
+            tokens.push(convertTemplatePart(this._tokens, this._code));
+            this._tokens = [];
+        };
```

Comment:
Small refactor to avoid disabling `no-underscore-dangle`, and I think this is a common way to avoid `that = this` since ES6.